dataactvalidator/config/sqlrules/a30_appropriations.sql

```diff
@@ -0,0 +1,16 @@
+SELECT approp.row_number,
+	approp.allocation_transfer_agency,
+	approp.agency_identifier,
+	approp.beginning_period_of_availa,
+	approp.ending_period_of_availabil,
+	approp.availability_type_code,
+	approp.main_account_code,
+	approp.sub_account_code
+FROM appropriation AS approp
+WHERE approp.submission_id = {}
+	AND NOT EXISTS (
+		SELECT 1
+		FROM object_class_program_activity AS op
+		WHERE approp.tas IS NOT DISTINCT FROM op.tas
```

Comment:
@cmc333333 Welcome to our vortex o' `NULL`s. You are now an official member of the team.

Lemme get your thoughts on this:
1. `tas` is required in both tables. If either or both has a blank `tas`, that will be reported by the required field checks that happen earlier in the validation process
2. we have a requirement not to report a pile of downstream errors that result from missing data. 
3. so I think Mike's idea here is to not produce an error record when either or both `tas`'s are `NULL` because we're already reporting that as missing data